### ISSUE TITLE
Readonly inputs and after element html in the backend

### ISSUE THREAD

**Issue by tzyganu** [magento/magento2#691]
Let's say I have a custom form that contains one input marked as readonly and has an `after_element_html` value.  

```
 $fieldset->addField('some_id', 'text', array(
     'name' => 'some_name',
     'label' => __('Some Label Here'),
     'readonly' => true,
     'after_element_html' => 'AFTER HTML'
));
```

The result would be something like this:
![After html](http://i.imgur.com/a6YUAbN.png)

The border on the element is continued with a border on whats after the element. But the borders don't match.

But if instead of `'readonly' => true,` I use `'disabled' => true,` then they match.  
So I guess the readonly css should look like the disabled css.

And yes...this is very low priority, but it would be nice to have a consistent UI.

**Comment by magento-engcom-team** [comment]
Hi @tzyganu. Thank you for your report. 
The issue has been fixed in magento/magento2#25611 by @drpayyne in 2.3-develop branch
Related commit(s):
 - [0e303660dbe08418c6653059dca7fa85cdf8f770](https://github.com/magento/magento2/commit/0e303660dbe08418c6653059dca7fa85cdf8f770)
 - [1298766deffe2c992b3f29aeb85454cc7bc6e44c](https://github.com/magento/magento2/commit/1298766deffe2c992b3f29aeb85454cc7bc6e44c)
 - [41c95a21ff19503b3870748f0c5a8bbb5b6d3bdb](https://github.com/magento/magento2/commit/41c95a21ff19503b3870748f0c5a8bbb5b6d3bdb)

The fix will be available with the upcoming 2.3.5 release.

**Comment by drpayyne** [comment]
@magento-engcom-team That PR was for fixing issue 691 in adobe-stock-integration as mentioned in its description, and not this issue.
